src/axom/sidre/core/Group.hpp

```diff
@@ -1588,7 +1588,7 @@ class Group
   bool m_is_list;
 
   /// Character used to denote a path string passed to get/create calls.
-  static const char s_path_delimiter;
```

Comment:
same q: do we need to export group? That should take care of related static vars, maybe I am missing something.